src/st/cen/cen.h

```diff
@@ -9,6 +9,11 @@
 
 #define OVL_EXPORT(x) CEN_##x
 
+enum OVL_EXPORT(Palette) {
+    PAL_NONE = 0,
+    PAL_BREAKABLE_DEBRIS = 0x159,
+};
+
```

Comment:
Won't all `OVL_DRA` palettes (anything without the top bit set) be shared? This could be moved to `include` since I think it's universal.

Reply:
No, not all non-8XXX palettes will be shared, only at most 0x100-0x1FF.  That being said, there hasn't been a word either way.  I'm not against adjusting this however is best, the enum is just the last word I've gotten..